### DISCUSSION TITLE
Jackson-datatype-joda in Dropwizard 3

### DISCUSSION THREAD

**Discussion by irineuruiz** [dropwizard/dropwizard#7070] [Q&A] [answered]
Hello,
I don't know if this is the best place to ask, but here goes.

We are trying to update from dropwizard 2.1.6 to version 3.

But we still have some code that we can't remove now that use Joda-Time.
I already added jackson-datatype-joda to the project but I wanted to know where is the correct place to register its module in Dropwizard in order to work as in version 2.1.6?

**Comment by mattnelson** [comment, marked as answer]
for joda objects in your http entities
```java
public void run(final Configuration configuration, final Environment environment) {
  environment.getObjectMapper().registerModule(new JodaModule())
```

if you have have joda objects in a custom config in config.yaml, you would also need to register the module in the bootstrap object mapper
```java
public void initialize(final Bootstrap<?> bootstrap) {
  bootstrap.getObjectMapper().registerModule(new JodaModule())
```
